static/src/stylesheets-atomised/newHeader.scss

```diff
@@ -0,0 +1,4 @@
+.New-header {
```

Comment:
_maybe_ – it really needs to be used for real, but i'd imagined a more suit-like capitalised name for the module. we'll need to see how it writes (hence not officially opening this up)